.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: ['plugin:react/recommended', 'standard-with-typescript'],
```

Comment:
こちらの記事によると、`prettier` を使うようです。
https://www.sunapro.com/eslint-with-prettier/

```suggestion
  extends: ['standard-with-typescript', 'prettier'],
```